introduction.md

```diff
@@ -1,2 +1,10 @@
 Hello, I'm Rafa Pereira
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?